MAUI/Cartesian-Charts/Annotation.md

```diff
@@ -0,0 +1,659 @@
+            Text = "Pixel value"
+        };
+
+        chart.Annotations.Add(rectangle);
+
+        this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Annotation in pixel value in MAUI Chart](Annotation_image/Rectangle_with_pixel.png)
+
+## Adding annotation for multiple axes
+
+When there are multiple axes, annotations can be added to a particular axis using the [XAxisName]() and [YAxisName]() properties. This is demonstrated in the code snippet below:
+
+{% tabs %}
+
+ {% highlight xaml %}
+
+    <chart:SfCartesianChart>
+    ...
+
+    <chart:SfCartesianChart.YAxes>
+        <chart:NumericalAxis Minimum="0" Maximum="1"/>
+        <chart:NumericalAxis Name="YAxis" CrossesAt="{Static x:Double.MaxValue}"/>
+    </chart:SfCartesianChart.YAxes>
+
+    <chart:SfCartesianChart.Annotations>
+
+        <chart:EllipseAnnotation X1="2" X2="4" Y1="10" Y2="15" Text="Ellipse" YAxisName="YAxis"/>
+
+    </chart:SfCartesianChart.Annotations>
+
+    </chart:SfCartesianChart>
+
+ {% endhighlight %}
+
+{% highlight c# %}
+
+    SfCartesianChart chart = new SfCartesianChart();
+
+    NumericalAxis yAxis1 = new NumericalAxis();
+    yAxis1.Minimum = 0;
+    yAxis1.Maximum = 1;
+    chart.XAxes.Add(yAxis1);
+    NumericalAxis yAxis2 = new NumericalAxis();
+    yAxis2.Name = "YAxis";
+    yAxis2.CrossesAt = double.MaxValue;
+    chart.YAxes.Add(yAxis2);
+
+    var ellipse = new EllipseAnnotation()
+    {
+        X1 = 2,
+        Y1 = 10,
+        X2 = 4,
+        Y2 = 15,
+        Text = "Ellipse",
+        YAxisName = "YAxis"
+    };
+
+    chart.Annotations.Add(ellipse);
+
+    this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Annotation in multiple axes in MAUI Chart](Annotation_image/Annotation_with_multiple_axes.png)
+
+## Text annotation
+
+The [TextAnnotation]() is used to add simple text at specific points in the chart area using the [Text]() property.
+
+{% tabs %}
+
+ {% highlight xaml %}
+
+    <chart:SfCartesianChart>
+    ...
+
+        <chart:SfCartesianChart.Annotations>
+        
+            <chart:TextAnnotation X1="2" Y1="25" Text="Text Annotation"/>
+            
+        </chart:SfCartesianChart.Annotations>
+
+    </chart:SfCartesianChart>
+ {% endhighlight %}
+
+{% highlight c# %}
+
+    SfCartesianChart chart = new SfCartesianChart();
+
+    var text = new TextAnnotation()
+    {
+        X1 = 2,
+        Y1 = 25,
+        Text = "Text Annotation"    
+    };
+
+    chart.Annotations.Add(text);
+
+    this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Text annotation in MAUI Chart](Annotation_image/Text_annotation.png)
+
+## Text Customization
+
+The [TextAnnotation]() can be customized using the [LabelStyle]() property. The [LabelStyle]() property provides options to customize the font-family, font-size, font-attributes and text color of axis labels. The following properties are used to customize the text:
+
+* [TextColor]() - Gets or sets the color for the text of the label.
```

Comment:
Gets or sets the text color for the label.

Reply:
changed